iCubTelemVizServer/https/conf/server.key

```diff
@@ -0,0 +1,15 @@
+-----BEGIN RSA PRIVATE KEY-----
+MIICXgIBAAKBgQDSEG3kwFvDoVAYtYyw8QdD/oU+gLOqYRXOPkaHOd0ifD+CWwOr
+2e0TStIvoXcLvTXq30DuvouNOKKC8YhEz8g2z6DIVGVjVPnCmtehRY214UfM3WxR
+dsD/91Lc5f4skTTGwSRTpfqQC0x8HmGyos1XL4D6nkTLqKTu39Vaw2wQ7QIDAQAB
+AoGBAMNM+Q77dJV+rdw7NrHvdaTWxwTkXaYjm0VkCuiAeMMuJ1HH0rL86vjL0Hzu
+LqMgaDeVF2agi1hh9l2tt9baI7Z441kA3COm3ptSvKL5qqSIYVX6ERYaCpxMNiXP
+HURAt9Qw8jUJHQ1WtZ8F/zRcrQ9m6BJnmCU8qBfXVry3kBKhAkEA/n4JvY3skRf4
+ah3cNZkgoUkjlnrxPSS1zrehWb5hr23wJewtUeEPApk/SMRL/rN8ov0OzxAI6BPN
+3hBis1zodQJBANNPAv0ERlJ3YNZBaDvTGK9BKjTADurmu/KoV1OvNsv+9TCWpSg0
+2o5IL2mOYiRJQ0d3X80HBR+snflGmvCkN5kCQQCLlMNw5OuwdO45/aCvAQqli4Dq
+Ywn3WDAjbhIgL8xSNME3e38r3n2mFjlghtGLpBQjLRq3NPs9lsceGeyYpceVAkAw
+mW6FhNMrqlsyS+7OO5nkzOcJJNcJfbHieIdFLvo6ZtQ7rZokqitOKBoHfADkeZko
+ubLl8+RgyZG3wF5oVEyZAkEA+qVeHarOnOvT1VKuR4EFrlhIuolLPP0M8QZvrYDj
+QHgrXWBuGUqR+2fAruWj9xjgTuL7qG0lGhuyzQ1XMFG3gw==
+-----END RSA PRIVATE KEY-----
```

Comment:
I would mention somewhere that this certificates and keys are stored in the repo as this is not an application that is supposed to be a real web application, as people otherwise may be confused by this, see https://stackoverflow.com/questions/35886794/should-files-involved-in-ssl-certificate-be-kept-confidential-added-to-gitigno .

Reply:
Good point! will do.

Reply:
As discussed T2T, we are using a websocket connection between YarpJS and Open MCT based on http, so we can just remove the respective server file and https folder with the generated keys.

I've opened an issue for tracking a later upgrade to https: https://github.com/dic-iit/element_software-engineering/issues/54.

Reply:
Removed unused https files.